src/database.rs

```diff
@@ -0,0 +1,63 @@
+extern crate rusqlite;
+
+pub struct StoredScript {
+    pub id: Option<i64>,
+    pub name: String,
+    pub source: String,
+    pub is_active: bool
+}
+
+pub struct ScriptDatabase {
+    connection: rusqlite::Connection
```

Comment:
We are going to use the database relatively seldom. I'm not sure we want to keep the connection opened all the time.